.github/workflows/test.yaml

```diff
@@ -5,9 +5,9 @@ on:
     # run this once a week (sunday midnight) to preserve cache
     # https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy
     - cron:  '0 0 * * 0'
-  pull_request_target:
+  pull_request:
     paths:
-      - '.github/workflows/test.yaml'
+      - '.github/**/*'
```

Comment:
`.github/**` would get every file in every subfolder, right? https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
I leaned on the behavior for bash, where a `**/*` would match every file in a sub-folder `.github` since we have them organized that way.
Would you rather match anything under `.github`?

Reply:
let's leave it as is!